### PR TITLE
[ES-393] [ES-508] esignet languages with unified login, transaction timeout error message

### PR DESCRIPTION
**Pull request by zesu22**

### CODE REVIEW

File: artifacts/src/i18n/esignet-i18n-bundle/ar.json [hunk 1]
```diff
@@ -120,13 +126,20 @@
...
     "login_linkName": "التسجيل المسبق",
     "more_ways_to_sign_in": "المزيد من الطرق لتسجيل الدخول",
     "invalid_auth_factor": "عوامل المصادقة غير صالحة. حاول مرة اخرى...",
     "download_now": "التحميل الان",
     "logo_alt": "e-Signet",
     "backgroud_image_alt": "قم بتسجيل الدخول باستخدام e-Signet",
-    "language": "لغة"
+    "language": "لغة",
+    "noAccount": "ليس لديك حساب؟",
+    "signup_with_one_login": "قم بالتسجيل باستخدام تسجيل الدخول الموحد"
```
Comment (ase-101) [inline artifacts/src/i18n/esignet-i18n-bundle/ar.json:138]: can we change key name also to signup_with_unified_login

File: artifacts/src/i18n/esignet-i18n-bundle/kn.json [hunk 2]
```diff
@@ -120,13 +126,20 @@
...
     "logo_alt": "ಇ-ಸಿಗ್ನೆಟ್",
     "backgroud_image_alt": "ಇ-ಸಿಗ್ನೆಟ್ ಮೂಲಕ ಸೈನ್ ಇನ್ ಮಾಡಿ",
-    "language": "ಭಾಷೆ"
+    "language": "ಭಾಷೆ",
+    "noAccount": "ಖಾತೆ ಇಲ್ಲವೇ?",
+    "signup_with_one_login": "ಏಕೀಕೃತ ಲಾಗಿನ್‌ನೊಂದಿಗೆ ಸೈನ್ ಅಪ್ ಮಾಡಿ"
+  },
+  "footer": {
+    "powered_by": "ನಡೆಸಲ್ಪಡುತ್ತಿದೆ",
+    "logo_alt": "அடிக்குறிப்பு லோகோ"
```
Comment (ase-101) [inline artifacts/src/i18n/esignet-i18n-bundle/kn.json:142]: Why is the value in tamil here?

Comment (zesu22) [reply artifacts/src/i18n/esignet-i18n-bundle/kn.json:142]: i will change that

Comment (zesu22) [reply artifacts/src/i18n/esignet-i18n-bundle/kn.json]: make the changes in [c0cf4c9](https://github.com/mosip/artifactory-ref-impl/pull/570/commits/c0cf4c9e23abc33549ca545f46d91cff31d808c4)

**Review by aranaravi — APPROVED** [review]

**Pull request merged**